lib/_base_viewmodels.dart

```diff
@@ -115,24 +104,29 @@ class StackedStream<T> extends SingleDataSourceViewModel<T> {
   T transformData(T data) {
     return data;
   }
+
+  @override
+  void dispose() {
+    _streamSubscription.cancel();
+    onCancel();
+    super.dispose();
+  }
 }
 
-abstract class StreamsViewModel extends BaseViewModel {
+abstract class MultiStreamViewModel extends BaseViewModel {
```

Comment:
This was a change that I was going to ask for after reading the first set of changes. I like the naming updates